.github/workflows/rawls-run-azure-e2e-tests.yaml

```diff
@@ -83,14 +94,16 @@ jobs:
       contents: 'read'
       id-token: 'write'
     outputs:
-      custom-version-json: ${{ steps.render-rawls-version.outputs.custom-version-json }}
+      custom-version-json: ${{ inputs.build-branch && steps.render-rawls-version.outputs.custom-version-json || ''}}
```

Comment:
The workflow we call supports an empty string to indicate that there are no custom overrides.